type-tests/files/createAction.typetest.ts

```diff
@@ -171,6 +173,38 @@ function expectType<T>(p: T): T {
 
   // typings:expect-error
   expectType<string>(strLenMetaAction('test').meta)
+  // typings:expect-error
+  expectType<boolean>(strLenMetaAction('test').error)
+}
+
+/*
+ * Test: adding boolean error with prepareAction
+ */
+{
+  const boolErrorAction = createAction('boolError', (payload: string) => ({
+    payload,
+    error: true
+  }))
+
+  expectType<boolean>(boolErrorAction('test').error)
+
+  // typings:expect-error
+  expectType<string>(boolErrorAction('test').error)
```

Comment:
from here on it's fine - that's excatly what we want to test here :)

Reply:
sounds good!